packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.controller.js

```diff
@@ -0,0 +1,73 @@
+import get from 'lodash/get';
+
+export default class {
+  /* @ngInject */
+  constructor(
+    $anchorScroll,
+    $location,
+    $translate,
+    Alerter,
+    datacenterBackupService,
+  ) {
+    this.$anchorScroll = $anchorScroll;
+    this.$location = $location;
+    this.$translate = $translate;
+    this.Alerter = Alerter;
+    this.backup = {};
+    this.alerter = Alerter;
+    this.datacenterBackupService = datacenterBackupService;
+  }
+
+  $onInit() {
+    this.alertMain = 'app.dedicatedClouds.datacenter.backup';
+    this.loader = {
+      updatingCapabilities: false,
+    };
+  }
+
+  scrollToTop() {
+    this.$location.hash('backupMainAlert');
+    this.$anchorScroll();
+  }
+
+  updateBackupCapabilities() {
+    this.loader.updatingCapabilities = true;
+    const capabilities = {
+      backupDurationInReport: this.backup.backupDurationInReport,
+      backupSizeInReport: this.backup.backupSizeInReport,
+      diskSizeInReport: this.backup.diskSizeInReport,
+      fullDayInReport: this.backup.fullDayInReport,
+      restorePointInReport: this.backup.restorePointInReport,
+      mailAddress: this.backup.mailAddress,
+      backupOffer: this.backup.backupOffer,
+    };
+    this.datacenterBackupService
+      .updateBackupCapabilities(this.productId, this.datacenterId, capabilities)
```

Comment:
Why not just pass `this.backup` ?

Reply:
These information are not available in the backup object

Reply:
`capabilities` is being passed to the API as such. Passing `this.backup` would result in a API call with data more that what is required

Reply:
Ok then maybe we could `_.pick` the properties needed from `this.backup` instead

Reply:
This is done. Thanks